n4js-libs/packages/n4js-cli/bin/PrintJavaVersion.java

```diff
@@ -0,0 +1,18 @@
+/**
+ * Copyright (c) 2020 NumberFour AG.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   NumberFour AG - Initial API and implementation
+ */
+
+/** Utility to detect and print out the Java version */
+public class PrintJavaVersion {
+	public static void main(String[] args) {
+		String versionStr = System.getProperty("java.version");
```

Comment:
This could also be dependent on the JRE / JDK impl.
You may want to use `java.lang.Runtime.version().feature()`

Reply:
Ok, Thanks!